fixture-monkey/src/test/java/com/navercorp/fixturemonkey/test/InnerSpecTest.java

```diff
@@ -136,7 +136,7 @@ void keyNullThrows() {
 	@Property
 	void keyInKey() {
 		FixtureMonkey sut = FixtureMonkey.builder()
-			.defaultArbitraryContainerInfo(new ArbitraryContainerInfo(1, 3))
+			.defaultArbitraryContainerInfoGenerator(ContainerPropertyGeneratorContext::getContainerInfo)
```

Comment:
You should not map getContaienrInfo since it is a result of a `size` manipulation.
If there is no size manipulation, throws NPE

Reply:
Do you have some clues for [this CI result](https://github.com/naver/fixture-monkey/actions/runs/4229897781/jobs/7346691308)?

CI with JDK 11 passed but not JDK 8 🤔